backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/impl/InventoryLineModularContractHandler.java

```diff
@@ -62,7 +62,8 @@
 @Component
 public class InventoryLineModularContractHandler implements IModularContractTypeHandler<I_M_InventoryLine>
 {
-	private static final AdMessageKey MSG_REACTIVATE_NOT_ALLOWED = AdMessageKey.of("de.metas.contracts.modular.impl.InventoryLineModularContractHandler.ReactivateNotAllowed");
+	private static final AdMessageKey MSG_ERROR_DOC_ACTION_NOT_ALLOWED = AdMessageKey.of("de.metas.contracts.DocActionNotAllowed");
+	private static final AdMessageKey MSG_ERROR_DOC_ACTION_UNSUPPORTED = AdMessageKey.of("de.metas.contracts.DocActionUnsupported");
```

Comment:
please depcleare these `AdMessageKey` constants at one place and use them everywhere they are needed.
(not in each single handler)

Reply:
done